python/destinations/S3/library.json

```diff
@@ -0,0 +1,133 @@
+            "Type": "EnvironmentVariable",
+            "InputType": "FreeText",
+            "Description": "Message count in the batch (0 saves data as they arrive, -1 disables batching based on message count)",
+            "DefaultValue": "0",
+            "Required": false
+        },
+        {
+            "Name": "parameter",
+            "Type": "EnvironmentVariable",
+            "InputType": "FreeText",
+            "Description": "Name of the parameter to save to S3",
+            "DefaultValue": "",
+            "Required": true
+        },
+        {
+            "Name": "aws_access_key_id",
+            "Type": "EnvironmentVariable",
+            "InputType": "HiddenText",
+            "Description": "AWS S3 access key id",
+            "DefaultValue": "",
+            "Required": false
+        },
+        {
+            "Name": "aws_access_key",
+            "Type": "EnvironmentVariable",
+            "InputType": "HiddenText",
+            "Description": "AWS S3 access key",
+            "DefaultValue": "",
+            "Required": false
+        },
+        {
+            "Name": "s3_bucket",
+            "Type": "EnvironmentVariable",
+            "InputType": "FreeText",
+            "Description": "AWS S3 bucket name",
+            "DefaultValue": "",
+            "Required": true
+        },
+        {
+            "Name": "prefix",
+            "Type": "EnvironmentVariable",
+            "InputType": "FreeText",
+            "Description": "File name prefix",
+            "DefaultValue": "",
+            "Required": false
+        },
+        {
+            "Name": "s3_folder",
+            "Type": "EnvironmentVariable",
+            "InputType": "FreeText",
+            "Description": "Name of the S3 folder to save to",
+            "DefaultValue": "",
+            "Required": false
+        },
+        {
+            "Name": "s3_folder_per_stream",
+            "Type": "EnvironmentVariable",
+            "InputType": "FreeText",
+            "Description": "Flag to save different streams to different S3 folders.",
+            "DefaultValue": "false",
+            "Required": false
+        },
+        {
+            "Name": "suffix",
+            "Type": "EnvironmentVariable",
+            "InputType": "FreeText",
+            "Description": "File suffix (e.g. file type extension)",
+            "DefaultValue": ".csv",
+            "Required": false
+        },
+        {
+            "Name": "timezone",
+            "Type": "EnvironmentVariable",
+            "InputType": "FreeText",
+            "Description": "Timezone (e.g. Asia/Singapore) for orchestrating time-based batching and file naming.",
+            "DefaultValue": "",
+            "Required": true
+        }
+    ],
+    "DeploySettings": {
+        "DeploymentType": "Service",
+        "CpuMillicores": 200,
+        "MemoryInMb": 200,
+        "Replicas": 1,
+        "PublicAccess": false,
+        "ValidateConnection": true
```

Comment:
You need to `print("CONNECTED!")` when the connection to the other system is established. So that the UI knows to navigate to the next screen. See the kinesis connector

Reply:
Alternatively don't validate connection

Reply:
72bd817d52a92411ba5d7db58b707c35e779aa23